plugin.rb

```diff
@@ -58,9 +58,29 @@ def self.sync!
     mount ::Kolide::Engine, at: '/kolide'
   end
 
+  User.register_custom_field_type "kolide_ip_addresses", [:string]
+
   reloadable_patch do |plugin|
     require_dependency 'user'
     class ::User
+      after_commit :update_kolide_ip_addresses, on: [:create, :update]
+
+      IP_ADDRESSES_FIELD = 'kolide_ip_addresses'
+
+      def update_kolide_ip_addresses
+        current_ip = ip_address&.to_s
+        return if current_ip.blank?
```

Comment:
won't `ip_address.changed?` work here?

Reply:
`ip_address`is a column and we're checking it in `after_commit`. `saved_change_to_ip_address?` can work here. Still, I want to check the `blank?`. Bot users will never have an IP.